guides/common/modules/proc_enabling-remote-execution.adoc

```diff
@@ -1,26 +1,25 @@
 :_mod-docs-content-type: PROCEDURE
 
-[id="enabling-remote-execution_{context}"]
+[id="enabling-remote-execution"]
 = Enabling remote execution
 
-Use this procedure to enable remote execution on your {SmartProxyServer}.
-To learn more about remote execution, see {ManagingHostsDocURL}Configuring_and_Setting_Up_Remote_Jobs_managing-hosts[Configuring and Setting Up Remote Jobs] in _{ManagingHostsDocTitle}_.
+Use this procedure to enable remote execution on your {SmartProxy}.
 
 ifdef::foreman-el,foreman-deb[]
 .Prerequisites
-* You have enabled the remote execution plugin on your {ProjectServer}.
-To do this, run the following command:
-+
-[options="nowrap", subs="+quotes,verbatim,attributes"]
-----
-# {foreman-installer} --enable-foreman-plugin-remote-execution
-----
+. Ensure that the remote executation plugin is enabled on your {ProjectServer}.
+For more information, see {ManagingHostsDocURL}installing-the-remote-execution-plugin[Installing the Remote Execution plugin] in _{ManagingHostsDocTitle}_.
 endif::[]
 
 .Procedure
-* Enable remote execution with `{foreman-installer}`:
+* On your {SmartProxy}, enable remote execution:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {foreman-installer} --enable-foreman-proxy-plugin-remote-execution-script
+# {foreman-installer} \
+--enable-foreman-cli-remote-execution \
```

Comment:
I wasn't implying we need to add this option here. We typically don't install hammer on proxies. Adding this option will probably fail for most users (it requires Hammer to be enabled, which has mandatory options IIRC) so I'd leave it out.